plugins/cef/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
```

Comment:
One additional note that just came to my mind: This also needs to be added to the CI matrix of the VAST Plugins job in the VAST workflow.

Just commenting here so we have something that can be resolved.

Reply:
Did 901a277bc do the trick?